### DISCUSSION TITLE
Előadáson elhangzott 2

### DISCUSSION THREAD

**Discussion by MicraMetripond** [nav-gov-hu/eRECEIPT#21] [Kérdések és válaszok]
1. előadás 20. dia: Mikor és milyen funkciók működését takarja az "AE 48 órás működése"?  A jelenlegi szabályzás szerint "8. Az AEE-nek alkalmasnak kell lennie arra, hogy a külső áramellátás megszűnését követően 2 órán keresztül adatkommunikációt folytasson, valamint a külső áramellátás megszűnését követően az adómegállapításhoz való jog elévülési idején belül megőrizze a benne tárolt adatokat"

**Comment by renced42** [comment]
Kedves @MicraMetripond 
A 48 órás működés során minden olyan funkciót biztosítani kell amikor normál működés van, vagyis normális az áramellátás. Ez a jelenlegi működéshez képes nagyobb időintervallum.
